docs/language/expressions.rst

```diff
@@ -114,8 +110,8 @@ a bool.
 Increment and Decrement operators
 _________________________________
 
-The post-increment and pre-increment operators are implemented like you would expect. So, ``a++``
-evaluates to the value of ``a`` before incrementing, and ``++a`` evaluates to value of ``a``
+The post-increment and pre-increment operators are implemented like you would expect. So, ``i++``
+evaluates to the value of ``i`` before incrementing, and ``++i`` evaluates to value of ``i``
```

Comment:
```suggestion
The post-increment and pre-increment operators are implemented by reading the variable's value before modifying it. ``i++``
evaluates to the value of ``i`` before incrementing, and ``++i`` evaluates to value of ``i``
```

`like you would expect` is not a clear way to say this. People have different understanding (sometimes wrong) about how things work.